app/handlers.py

```
@@ -61,3 +61,14 @@ def invalid_token_callback(error):
         """Get backend version."""
         return {"status": "success", "data": Config.VERSION}, status.HTTP_200_OK
+
+    from werkzeug.exceptions import HTTPException
+
+    @app.errorhandler(Exception)
+    def handle_error(e):
+
+        if isinstance(e, HTTPException):
+            return e
+
+        current_app.logger.exception(e)
```

Ruff (0.15.9)

[error] Undefined name `current_app`

(F821)